tests/sample/unsafe/droppacket_unsafe.c

```diff
@@ -31,7 +31,7 @@ struct
 
 SEC("xdp")
```

Comment:
```suggestion
SEC("xdp_test")
```

Reply:
We had an internal discussion.  This is a good catch (xdp to xdp_test). It is a miss during the initial commit of #2974.   This PR is targeted for minimal changes so the xdp-for-windows can declare XDP headers and hooks, and provide a nupkg. Hence not changing it.